src/resolve.c

```diff
@@ -1764,7 +1764,8 @@ resolve_schema_nodeid(const char *nodeid, const struct lys_node *start, const st
 
     /* descendant-schema-nodeid */
     if (is_relative) {
-        cur_module = start_mod = start->module;
+        // cur_module = start_mod = start->module;
+        cur_module = start_mod = lys_main_module(start->module);
```

Comment:
Please rather use `lys_node_module(start)`, it is meant for these cases, thanks.

Reply:
done

Reply:
Unfortunately the imp_aug branch was not clean...
Next attempt.

Reply:
from my point of view, it's through. I don't know what codecov is complaining about, I guess, it's because of the removed test case (which was not intended to be part of the pull request) . Does it compare to the previous build?